Makefile

```diff
@@ -44,19 +58,35 @@ tags:
 	#find chroma-agent/chroma_agent chroma-manager/{tests,chroma_{agent_comms,api,cli,core,ui}} -type f | ctags -L -
 	ctags --python-kinds=-i -R --exclude=chroma-manager/_topdir --exclude=chroma-\*/myenv\* --exclude=chroma-manager/chroma_test_env --exclude=chroma-dependencies --exclude=chroma_unit_test_env --exclude=chroma-manager/ui-modules .
 
+fetch_build:
+	curl -Lk -o $(ARCHIVE_PATH) $(BUILD_URL)/$(ARCHIVE_FILENAME)
+	# fetch integration tests package (not included in iml release archive)
+	if [ ! -z "$(DEPS_URL)" ]; then                                               \
+		set -e;                                                                   \
+		FILENAME=$$(curl -k --list-only $(DEPS_URL)/repo/ | grep chroma-manager-integration | sed -e 's/.*\(chroma-manager-integration-[^/]*\)\/.*/\1/'); \
+		echo $$FILENAME;                                                          \
+		mkdir -p chroma-manager/dist;                                             \
+		curl -Lk -o chroma-manager/dist/$$FILENAME $(DEPS_URL)/repo/$$FILENAME;   \
+	fi
+
 # build the chroma-{agent,management} subdirs before the chroma-dependencies subdir
 chroma-dependencies: chroma-agent chroma-manager chroma-diagnostics
 chroma-bundles: chroma-dependencies
 
 destroy_cluster: Vagrantfile
-	vagrant destroy -f
-	sed -ie '/# VAGRANT START/,/# VAGRANT END/d' ~/.ssh/config
-	sed -ie '/IML Vagrant cluster/d' ~/.ssh/authorized_keys
-	export LIBVIRT_DEFAULT_URI=qemu:///system;                       \
-	for net in intel-manager-for-lustre{0,1,2,3} vagrant-libvirt; do \
-	    virsh net-destroy $$net || true;                             \
-	    virsh net-undefine $$net || true;                            \
-	done
+	set -e;                                                              \
+	[[ -e $(VAGRANT_VM3_LIBVIRT_DIR)/id ]] && LIBVIRT="1";               \
+	vagrant destroy -f;                                                  \
+	sed -ie '/# VAGRANT START/,/# VAGRANT END/d' ~/.ssh/config;          \
+	sed -ie '/IML Vagrant cluster/d' ~/.ssh/authorized_keys;             \
+	if [ ! -z "$$LIBVIRT" ]; then                                        \
```

Comment:
Why not just use `[ -e $(VAGRANT_VM3_LIBVIRT_DIR)/id ]` instead of setting `LIBVIRT="1"` earlier, which is only used here, just to be consistent with how you use `[ -e $(VAGRANT_VM3_LIBVIRT_DIR)/id ]` in the `create_cluster` recipe?

Reply:
because at that point , it wouldn't be set (it's after the destroy command)

Reply:
Heh.  Of course.  ~smacks forehead~